<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XMachine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0">
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_local">
          <property name="title">
           <string>本地网络配置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_local_in" stretch="0,1">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>内网</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_in"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_local_0" stretch="0,1,0,0">
             <item>
              <widget class="QLabel" name="label_output_1">
               <property name="text">
                <string>通道0 IP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_local_ip_0"/>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>端口</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_local_port_0">
               <property name="text">
                <string>10000</string>
               </property>
               <property name="maxLength">
                <number>6</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_local_1" stretch="0,1,0,0">
             <item>
              <widget class="QLabel" name="label_output_2">
               <property name="text">
                <string>通道1 IP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_local_ip_1"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>端口</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_local_port_1">
               <property name="text">
                <string>10000</string>
               </property>
               <property name="maxLength">
                <number>6</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_remote">
          <property name="title">
           <string>远端网络配置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_remote_0" stretch="0,1,0,0">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>通道0 IP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_remote_ip_0">
               <property name="text">
                <string>192.168.2.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>端口</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_remote_port_0">
               <property name="text">
                <string>10000</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_remote_1" stretch="0,1,0,0">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>通道1 IP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_remote_ip_1">
               <property name="text">
                <string>192.168.2.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>端口</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_remote_port_1">
               <property name="text">
                <string>10000</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_control">
        <property name="title">
         <string>控制</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_updateInterface">
           <property name="text">
            <string>刷新本地网卡列表</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_start">
           <property name="text">
            <string>启动</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_stop">
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>106</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
